straxen/plugins/peaklet_processing.py

```diff
@@ -717,6 +723,8 @@ def compute(self, peaklets, lone_hits):
                 max_buffer=int(self.config['s2_merge_max_duration']//np.gcd.reduce(peaklets['dt'])),
             )
             merged_s2s['type'] = 2
+            merged_s2s['s2_prob'] = 0 # remember this is ln prob so 0 means 100%
```

Comment:
Any reason you keep the ln of the probability instead of converting it to actual probability? Obviously for computing it with the classifier you prefer to use the ln which is fine but why keep it as ln when you are naming everything without the ln? this can produce unnecessary confusion so do you gain anything later down in the chain from not taking the exponent after its calculated?

Reply:
We hose to return these ln probabilities rather than the probabilities because of floating point errors / computer rounding.

Reply:
A double precision floating point is accurate to ~15 decimal points. Naive bayes uses the product of the empiric probability of features ie the relative frequency in the labeled dataset. Are you telling me that you expect to have a labeled dataset with 10^15 waveforms to reach the level of confidence in your probability estimate that you are worried about the 15th decimal being rounded?